src/pages/settings/index.vue

```diff
@@ -16,6 +16,11 @@
           <span class="settings-menu__item-title"> {{ $t('settings_following') }}</span>
         </NuxtLink>
       </li>
+      <li>
+        <NuxtLink class="settings-menu__item" :to="{ name: 'settings-notification' }">
+          <span class="settings-menu__item-title"> {{ $t('settings_notification') }}</span>
+        </NuxtLink>
+      </li>
```

Comment:
Can merge with `email` page, setting email and notification seems highly related